src/complex/DataStructure/DataStore.hpp

```diff
@@ -79,6 +79,11 @@ class DataStore : public IDataStore<T>
 
   ~DataStore() override = default;
 
+  T* data() const
+  {
+    return m_Data.get();
+  }
+
```

Comment:
Is this used as part of the DataGroup creation? If not I can just commit it to master instead.

Reply:
Sure. I need the API to more efficiently read files directly into a DataStore<T> instance.